pkg/types/installconfig.go

```diff
@@ -31,6 +32,7 @@ var (
 		azure.Name,
 		gcp.Name,
 		openstack.Name,
+		ovirt.Name,
```

Comment:
In order to disable TUI (so this becomes install-config only) you can just move this line from here (`PlatformNames`) to `HiddenPlatformNames`. We can keep all of the TUI/survey code in the PR. This just acts as a flag on whether it is presented to user.

Reply:
That's fine with me.

Reply:
> We can keep all of the TUI/survey code in the PR. This just acts as a flag on whether it is presented to user.

Actually wizard code should be removed and included in a separate PR. `ovirt.Name` should still be included in `HiddenPlatformNames`